CHANGELOG.md

```diff
@@ -17,6 +18,9 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a Ch
 
 ### Breaks
 * Removed support for Python 3 prior to 3.6
+* Removed `get_changes`
```

Comment:
Complete agree the function can be greatly simplified, but its an internal function, so im not sure this constitutes a breaking change to our users. (we dont exactly document use of the utility class in our docs)

Reply:
100%. I just wasn't sure what constitutes `mc706/changelog-cli`'s public interface.